theorems/T000582.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: T000582
+if:
+  and:
+  - P000187: true
+  - P000003: true
+  - P000197: true
+then:
+  P000191: true
+refs:
+  - doi: 10.1016/0016-660X(76)90024-6
+    name: Infinite games and generalizations of first-countable spaces (Gruenhage)
+---
+
+Shown in the Theorem 3.7 of {{doi:10.1016/0016-660X(76)90024-6}}.
```

Comment:
I expect this is correct, but I think we need a few more details here.

Reply:
Alright, I have just added more details in its description.